src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetDAO.java

```diff
@@ -185,6 +200,29 @@ public void insertOrUpdateAktivitetStatus(List<AktivitetStatus> aktivitetStatuse
                 .collect(toList());
     }
 
+    public List<ArenaAktivitetDTO> hentUtgatteAktivteter(String aktivitetsType) {
+        String sql = "SELECT " + AKTIVITETID + ", " + AKTIVITETID + " FROM " + TABLE_NAME
```

Comment:
Is this mistake, I suppose one is aktoerid.

Reply:
Nice catch!